sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -122,72 +121,6 @@ abstract class SessionCatalogSuite extends AnalysisTest with Eventually {
     assert(e.contains(s"`$name` is not a valid name for tables/databases."))
   }
 
-  test("create table with default columns") {
-    def test: Unit = withBasicCatalog { catalog =>
-      assert(catalog.externalCatalog.listTables("db1").isEmpty)
-      assert(catalog.externalCatalog.listTables("db2").toSet == Set("tbl1", "tbl2"))
-      catalog.createTable(newTable(
-        "tbl3", Some("db1"), defaultColumns = true), ignoreIfExists = false)
-      catalog.createTable(newTable(
-        "tbl3", Some("db2"), defaultColumns = true), ignoreIfExists = false)
-      assert(catalog.externalCatalog.listTables("db1").toSet == Set("tbl3"))
-      assert(catalog.externalCatalog.listTables("db2").toSet == Set("tbl1", "tbl2", "tbl3"))
-      // Inspect the default column values.
-      val db1tbl3 = catalog.externalCatalog.getTable("db1", "tbl3")
-      val currentDefault = ResolveDefaultColumns.CURRENT_DEFAULT_COLUMN_METADATA_KEY
-
-      def findField(name: String, schema: StructType): StructField =
-        schema.fields.filter(_.name == name).head
-      val columnA: StructField = findField("a", db1tbl3.schema)
-      val columnB: StructField = findField("b", db1tbl3.schema)
-      val columnC: StructField = findField("c", db1tbl3.schema)
-      val columnD: StructField = findField("d", db1tbl3.schema)
-      val columnE: StructField = findField("e", db1tbl3.schema)
-
-      val defaultValueColumnA: String = columnA.metadata.getString(currentDefault)
-      val defaultValueColumnB: String = columnB.metadata.getString(currentDefault)
-      val defaultValueColumnC: String = columnC.metadata.getString(currentDefault)
-      val defaultValueColumnD: String = columnD.metadata.getString(currentDefault)
-      val defaultValueColumnE: String = columnE.metadata.getString(currentDefault)
-
-      assert(defaultValueColumnA == "42")
-      assert(defaultValueColumnB == "\"abc\"")
-      assert(defaultValueColumnC == "_@#$%")
-      assert(defaultValueColumnD == "(select min(x) from badtable)")
-      assert(defaultValueColumnE == "41 + 1")
-
-      // Analyze the default column values.
-      val statementType = "CREATE TABLE"
-      assert(ResolveDefaultColumns.analyze(columnA, statementType).sql == "42")
```

Comment:
no need to test this specific analyzer any more. The default expression will be resolved with the main analyzer with the commands.